.github/workflows/ci_msw.yml

```diff
@@ -113,6 +116,7 @@ jobs:
             msbuild /noLogo /m '/p:Configuration=${{ matrix.configuration }}' /p:Platform=${{ matrix.platform }} tests\test_vc16.sln
 
       - name: Run tests
+        if: ${{ !contains(matrix.platform, 'arm64') }}
```

Comment:
Do the tests fail under ARM or is there some other reason to exclude them?

Reply:
(I'm working through setting up and running the tests now, as yes, they should be passing for check-in.)

Reply:
With the `tests` folder as the working directory and DPI set to 100%, all tests pass for ARM64, ARM64EC, and x64 on ARM64 Windows 11 (26100). (Since the _gui tests rely on pixel counts, non-unity DPI breaks a number of them--but that isn't ARM64-specific and is easy to control for.)

[vc_x64_mswu_test.txt](https://github.com/user-attachments/files/15963715/vc_x64_mswu_test.txt)
[vc_x64_mswu_test_gui.txt](https://github.com/user-attachments/files/15963717/vc_x64_mswu_test_gui.txt)
[vc_arm64_mswu_test.txt](https://github.com/user-attachments/files/15963718/vc_arm64_mswu_test.txt)
[vc_arm64_mswu_test_gui.txt](https://github.com/user-attachments/files/15963719/vc_arm64_mswu_test_gui.txt)
[vc_arm64ec_mswu_test.txt](https://github.com/user-attachments/files/15963720/vc_arm64ec_mswu_test.txt)
[vc_arm64ec_mswu_test_gui.txt](https://github.com/user-attachments/files/15963721/vc_arm64ec_mswu_test_gui.txt)

Reply:
Thanks! Do you plan updating this PR to run the tests in the CI builds?

Reply:
Currently, no. It would require setting up and manually managing runners for the wxWidgets org and moving the tests to a second runner to execute--or just building ARM64 on an ARM64 runner then running the tests locally.

I'm happy to either a) adjust this PR to not touch CI at all until that's possible (or whatever scope you feel is appropriate) or b) wait until GH-hosted ARM64 Windows runners are available. (Technically [they're available now in beta](https://github.blog/2024-06-03-arm64-on-github-actions-powering-faster-more-efficient-build-systems/), but I'm not sure how much you'd have to do from the wxWidgets org perspective to set them up.)

Reply:
Thanks, I'll just (squash) merge this for now then (unless you prefer to rebase it to keep your commits).

Reply:
Sounds good to me; I prefer squash merges myself.

Reply:
Side note: if no one else beats me to it, I'm happy to come back later and work on setting up ARM64 testing once the GH-hosted runners are available. I'd probably move the ARM64 and ARM64EC builds to ARM64 so there's no shuffling around of generated artifacts. Once some confidence is had, it probably wouldn't be too hard to add ARM64 binaries to releases.

Reply:
This would be appreciated, TIA!